evap/evaluation/models.py

```diff
@@ -512,19 +520,35 @@ class TextAnswer(Answer):
     """A free-form text answer to a question (usually a comment about a course
     or a contributor)."""
 
-    elements_per_page = 5
-
     reviewed_answer = models.TextField(verbose_name=_("reviewed answer"), blank=True, null=True)
     original_answer = models.TextField(verbose_name=_("original answer"), blank=True)
-
-    checked = models.BooleanField(verbose_name=_("answer checked"), default=False)
-    hidden = models.BooleanField(verbose_name=_("hide answer"), default=False)
+    
+    TEXT_ANSWER_STATES = (
+        ('N', _('not published')),
+        ('P', _('published privately')),
+        ('Y', _('published')),
+        ('', _('not reviewed')),
+    )
+    state = models.CharField(max_length=1, choices=TEXT_ANSWER_STATES, verbose_name=_('state of answer'), default='')
 
     class Meta:
         verbose_name = _("text answer")
         verbose_name_plural = _("text answers")
 
     @property
+    def reviewed(self):
+        return self.state != ''
```

Comment:
i will change it to constants. however, i will not change other occurrences (namely the courses' statuses) in this pr - this would need to go somewhere else.